metron-interface/metron-rest/src/test/java/org/apache/metron/rest/controller/SensorParserConfigControllerIntegrationTest.java

```diff
@@ -196,11 +198,17 @@ public void test() throws Exception {
     cleanFileSystem();
     this.sensorParserConfigService.delete("broTest");
     this.sensorParserConfigService.delete("squidTest");
-
+    Method[] method = SensorParserConfig.class.getMethods();
+    int numFields = 0;
+    for(Method m : method) {
+      if(m.getName().startsWith("set")) {
```

Comment:
we should have a reflection utils method for getting these maybe.  Setters would also have to be checked for having one parameter and void return I think